modules/rest-http/src/main/java/org/apache/ignite/internal/visor/cache/VisorCacheGetValueTask.java

```diff
@@ -0,0 +1,203 @@
+import org.apache.ignite.internal.processors.task.GridInternal;
+import org.apache.ignite.internal.processors.task.GridVisorManagementTask;
+import org.apache.ignite.internal.util.typedef.internal.S;
+import org.apache.ignite.internal.visor.VisorJob;
+import org.apache.ignite.internal.visor.VisorOneNodeTask;
+import org.apache.ignite.internal.visor.query.VisorQueryUtils;
+import org.apache.ignite.internal.visor.util.VisorTaskUtils;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * Task that get value in specified cache for specified key value.
+ */
+@GridInternal
+@GridVisorManagementTask
+public class VisorCacheGetValueTask extends VisorOneNodeTask<VisorCacheGetValueTaskArg, VisorCacheModifyTaskResult> {
+    /** */
+    protected static final ObjectMapper MAPPER = new ObjectMapper();
+
+    /** */
+    private static final long serialVersionUID = 0L;
+
+    /** {@inheritDoc} */
+    @Override protected VisorCacheModifyJob job(VisorCacheGetValueTaskArg arg) {
+        return new VisorCacheModifyJob(arg, debug);
+    }
+
+    /**
+     * Job that get value in specified cache for specified key value.
+     */
+    private static class VisorCacheModifyJob extends VisorJob<VisorCacheGetValueTaskArg, VisorCacheModifyTaskResult> {
+        /** */
+        private static final long serialVersionUID = 0L;
+
+        /**
+         * Create job.
+         *
+         * @param arg Task argument.
+         * @param debug Debug flag.
+         */
+        private VisorCacheModifyJob(VisorCacheGetValueTaskArg arg, boolean debug) {
+            super(arg, debug);
+        }
+
+        /**
+         * Convert string value to value of specified type.
+         *
+         * @param type Expected type.
+         * @param value String presentation of value.
+         * @return Value of specified type.
+         */
+        private Object parseArgumentValue(VisorObjectType type, String value) {
+            switch (type) {
+                case CHARACTER:
+                    return value.charAt(0);
+
+                case INT:
+                    return Integer.valueOf(value);
+
+                case LONG:
+                    return Long.valueOf(value);
+
+                case SHORT:
+                    return Short.valueOf(value);
+
+                case BYTE:
+                    return Byte.valueOf(value);
+
+                case FLOAT:
+                    return Float.valueOf(value);
+
+                case DOUBLE:
+                    return Double.valueOf(value);
+
+                case BOOLEAN:
+                    return Boolean.valueOf(value);
+
+                case UUID:
+                    return UUID.fromString(value);
+
+                case TIMESTAMP:
+                    return new Timestamp(Long.parseLong(value));
+
+                case DATE:
+                    return new Date(Long.parseLong(value));
+
+                case BINARY:
+                    try {
+                        JsonNode obj = MAPPER.readTree(value);
```

Comment:
Please define wrapper class and map to it:
{
  className,
  fields: [...]
}

Reply:
Wrapper implemented.